src/invert/fft_fftw.cxx

```diff
@@ -108,10 +142,7 @@ void rfft(MAYBE_UNUSED(const BoutReal *in), MAYBE_UNUSED(int length), MAYBE_UNUS
      */
     fout = static_cast<fftw_complex*>(fftw_malloc(sizeof(fftw_complex) * (length/2 + 1)));
 
-    unsigned int flags = FFTW_ESTIMATE;
-    if (fft_measure) {
-      flags = FFTW_MEASURE;
-    }
+    auto flags = get_flags(fft_flag);
```

Comment:
This is much nicer than what we had before, even without the extra flag. Thanks!